scripts/globals/combat/physical_utilities.lua

```diff
@@ -565,18 +565,21 @@ end
 -- Fencer: Critical hit rate bonus when actor is only wielding with main hand.
 xi.combat.physical.criticalRateFromFencer = function(actor)
     local fencerBonus = 0
-
-    local mainEquip = actor:getStorageItem(0, 0, xi.slot.MAIN)
-    local subEquip  = actor:getStorageItem(0, 0, xi.slot.SUB)
+    -- TODO: do any Trusts or mobs ever get Fencer bonuses?
 
     if
-        actor:getObjType() == xi.objType.PC and
-        mainEquip and
-        not mainEquip:isTwoHanded() and                                                      -- No 2 handed weapons.
-        not mainEquip:isHandToHand() and                                                     -- No 2 handed weapons.
-        (subEquip == nil or subEquip:getSkillType() == xi.skill.NONE or subEquip:isShield()) -- Only shields allowed in sub.
+        actor:getObjType() == xi.objType.PC
     then
```

Comment:
if only 1 condition is left in this if, can it be all 1 line?

Reply:
it sure can, I just move dit "up" and didn't think about it. Might look a little funny though!

Reply:
change made 👍